src/main/java/com/basho/riak/client/core/operations/ts/StoreOperation.java

```diff
@@ -59,12 +46,13 @@ public String getQueryInfo()
 
     private String createQueryInfoMessage()
     {
-        return "INSERT <" + this.rowCount + " rows> into " + this.tableName;
+        return "INSERT into " + builder.tableName;
```

Comment:
Kinda. Every operation implements a "getQueryInfo" method, that is supposed to give a succinct description of what is happening with the operation/command. The "query" vocabulary just overlaps here.